stable/spinnaker/templates/statefulsets/halyard.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-halyard
+  labels:
+    app: {{ template "fullname" . }}
+    component: halyard
+spec:
+  ports:
+  - port: 8064
+    name: daemon
+  clusterIP: None
+  selector:
+    app: {{ template "fullname" . }}
+    component: halyard
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "fullname" . }}-halyard
+spec:
+  serviceName: {{ template "fullname" . }}-halyard
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+      component: halyard
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        component: halyard
+    spec:
+      securityContext:
+        runAsUser: 1000
+        fsGroup: 1000
+      initContainers:
+      - name: "create-halyard-local"
+        image: {{ .Values.halyard.image.repository }}:{{ .Values.halyard.image.tag }}
+        command:
+        - bash
+        - -c
+        - "cp /opt/halyard/config/* /tmp/config && printf 'server:\n  address: 0.0.0.0\n' > /tmp/config/halyard-local.yml"
```

Comment:
`server.address: 0.0.0.0\n` might be a little cleaner

Reply:
Done! Good call.